code/go/0chain.net/smartcontract/zcnsc/mint_test.go

```diff
@@ -81,6 +81,22 @@ func Test_EmptySignaturesShouldFail(t *testing.T) {
 	require.Error(t, err)
 }
 
+func Test_EmptyAuthorizersShouldFail(t *testing.T) {
```

Comment:
add test where number of signatures are not empty

Reply:
Done